src/FirstSteps/RPG/Adventures/Hell.cs

```diff
@@ -0,0 +1,48 @@
+﻿using FirstSteps.RPG.Heroes;
+using FirstSteps.RPG.Items;
+using FirstSteps.RPG.Tools;
+using Spectre.Console;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Channels;
+
+namespace FirstSteps.RPG.Adventures
+{
+    public  class Hell
+    {
+        HeroesDamage heroes = new HeroesDamage();
+        public void FightWithboss(Hero hero)
+        {
+            Boss boss = new Boss("Diablo", 40, 5, 7);
+            Display.DefaultText($"You entered to the Hell!\nIf u wanna fight with {boss.Name} press 'f' otherwise go back to previous menu ");
+            Display.ErrorText("Boss stats:");
+            Display.ItemText($"\n Health = {boss.Health}\n Damage = {boss.Damage}\n Strenght = {boss.Strength}");
+
+
+            Console.WriteLine("Press 'z' to hit or run away press 'y'");
+            while (true)
```

Comment:
It's better to declare the end condition explicitly. You can check here if the hero ran away or died.